nixos/modules/services/misc/paperless.nix

```
@@ -439,5 +490,57 @@ in
         gid = config.ids.gids.paperless;
       };
     };
-  };
+  }
+
+  (lib.mkIf cfg.exporter.enable {
+    systemd.tmpfiles.rules = [
+      "d '${cfg.exporter.directory}' - ${cfg.user} ${config.users.users.${cfg.user}.group} - -"
+    ];
+
+    systemd.services.paperless-exporter = {
+      description = "paperless exporter service";
+
+      startAt = cfg.exporter.onCalendar;
```

```suggestion
      startAt = lib.defaultTo [] cfg.exporter.onCalendar;
```
Fixes error when `onCalendar = null`.

---

Or just simply mkIf it.